src/client/components/ParticipantDetailsPage/PresenceHistory.tsx

```diff
@@ -24,7 +23,16 @@ export const PresenceHistory: React.FC<PresenceHistoryProps> = (props) => (
           <tr key={time.toString()}>
             <td>{time.format('L [klo] LT')}</td>
             <td>{getPresenceLabel(row.presence)}</td>
-            <td>Käyttäjä #{row.authorId}</td>
+            <td>
+              {row.author ? (
+                <span>
+                  {row.author.firstName} {row.author.lastName}
+                </span>
+              ) : (
+                <span>Unknown user</span>
```

Comment:
The frontend language for now is finnish, not english.

Also, the spans on both sides of the ternary here are not really needed, you could just return plain strings

Reply:
Thanks, fixed